api-docs/list-clients.md

```diff
@@ -24,9 +24,10 @@ Notes:
       "id": 56,
       "firstName": "Mario",
       "lastName": "Luigi",
+      "fullName": "Mario Luigi",
       "zip": "84107",
       "birthday": "1981-01-01",
-      "primaryPhone": "5551111111",
+      "phone": "5551111111",
```

Comment:
To match [get client api](https://github.com/JustUtahCoders/comunidades-unidas-internal/blob/master/api-docs/get-client.md)